styles.css

```diff
@@ -30,7 +48,266 @@
 /* ------------------------------- 4. Areas ---------------------------------------- */
 
 /* --------------- 4.1 Header ------------------------- */
+.navbar-outer-container {
+    display: flex;
+    background: rgb(72,69,228);
+    background: linear-gradient(168deg, rgba(72,69,228,1) 0%, rgba(87,34,205,1) 35%);
+}
+
+
+.navbar-items-inner-container {
+    background-color: #5722CD;
```

Comment:
Het zou mooi zijn om de kleuren in CSS-variabelen te zetten en die dan te hergebruiken!